docs/proposals/rename-group.md

```diff
@@ -0,0 +1,164 @@
+
+### Enable the use of topolvm.cybozu.com
+
+We will make the following changes to allow users to continue to use `topolvm.cybozu.com`.
+
+#### Addition of group name setting function
+
+- If you set the `USE_LEGACY` environment variable, the group name `topolvm.cybozu.com` will be used
+- If the `USE_LEGACY` environment variable is not set, the group name `topolvm.io` will be used
+
+#### Group name setting in helm chart
+
+Add the `.Values.useLegacy` variable to the helm chart and set this variable to `true` to set the installation manifest to use `topolvm.cybozu.com`.
+If the `.Values.useLegacy` variable is not set to `true`, the group name will be `topolvm.io`.
+
+If you are going to set the following variables, you may need to set them appropriately to use `topolvm.io` or `topolvm.cybozu.com` in the variables.
+
+- `.Values.storageClasses`
+- `.Values.node.volumes`
+- `.Values.volumeMounts.topolvmNode`
+
+#### Automatic generation for API and CRD
+
+Automatically generating the API and CRD of `topolvm.cybozu.com` based on the API and CRD of `topolvm.io`.
+
+### Auto group conversion for LogicalVolume
+
+Add a wrap client for controller-runtime client to automatically convert a group of LogicalVolume resources.
+Change each controller and other components to use this wrapped client.
+
+### Things to be done by users of topolvm.cybozu.com for upgrading the helm chart
+
+Since this change was released, users of TopoLVM with the group name `topolvm.cybozu.com` will need to take the following actions when upgrading the helm chart.
+
+- Set the `.Values.useLegacy `variable to` true` in `values.yaml` in the helm chart
+- If there is a part of the `values.yaml` in the helm chart that uses the string `topolvm.cybozu.com`, change it to `topolvm.io`
+
+If you upgrade the helm chart without taking above actions, it is possible that TopoLVM related resources will be deleted.
+
+### Migrate from topolvm.cybozu.com to topolvm.io
+
+If you are already using TopoLVM with the group name `topolvm.cybozu.com` and want to migrate the group name to `topolvm.io` after the release of this change, you can manually change the data by the following method.
+
+1. Avoid booting pods using TopoLVM volumes during migration.
+1. Temporarily stop the following pods:
+   - topolvm-controller
+   - topolvm-node
+   - topolvm-scheduler
+1. Manually install the CRD for `topolvm.io`
+1. Perform the migration work for each resource as mentioned in the chapter `Changes to topolvm.io`.
+   - Please migrate while confirming that the pod using the updated resource (e.g. Persistent Volume) continues to operate without problems.
+1. If the migration of various data is successful, restart the stopped pod.
```

Comment:
This way looks not match current PR changes. We can deploy TopoLVM using a new group name on another namespace. Users should migrate user data and don't have to change group names manually.

Reply:
Thank you. I updated the migration process.
https://github.com/topolvm/topolvm/pull/539/commits/929e4f461d0cda5890910926abc2adb8c2428803